docs/ug/cli.md

```diff
@@ -107,6 +107,15 @@ The last modified dates will be in the same timezone specified with the `--timez
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
+### `--shallow-cloning`, `-S`
+
+**`--shallow-cloning`**: Clones repositories using Git's shallow cloning functionality, which can significantly reduce the time taken to clone large repositories. However, the flag should not be used for smaller repositories where the .git file is smaller than 500 MB, as it would create overhead. Note: this flag is incompatible with the `--last-modified-date` flag.
```

Comment:
Instead of `Note:` better to use the `MarkBind info box` we use throughout the page.